jorgePrecich/ruby/rb_terminal.rb

```diff
@@ -0,0 +1,19 @@
+if ARGV.length < 2
+  puts "You need to write at least 2 values -> ClassName, Attribute"
+else
+  class_name = ARGV.shift().capitalize
```

Comment:
In this case `capitalize` wouldn't be the correct method to use, for example, if you write the class name as `my_class` it would change it to `My_class` that is not correct to Ruby class names convention, it should be `MyClass`, so, I recommend using `snake_camel` gem, it will do the trick for you.

Install it using: `gem install 'snake_camel'`
Require it: `require 'snake_camel'`
and use it:
```ruby
my_class_name.camelcase
my_class_name.snakecase
```

> [Info about naming](https://github.com/rubocop-hq/ruby-style-guide#naming)

Reply:
You are right. I will add the gem to do the capitalization properly